repo/packages/G/gestalt-framework/1/resource.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images": {
+    "icon-large": "https://s3.amazonaws.com/gfi.misc/Gestalt-Icon-Large.png",
+    "icon-medium": "https://s3.amazonaws.com/gfi.misc/Gestalt-Icon-Medium.png",
+    "icon-small": "https://s3.amazonaws.com/gfi.misc/Gestalt-Icon-Small.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
```

Comment:
urgh, tough call. the framework comprises of multiple docker images, versioned ensemble. i can register them all in the `resource.json` file to facilitate on-prem docker registries. however, the universe documentation requests that these also use the version ID for the docker images, which means that we're forced to do a whole new release for a hotfix, instead of just moving a tag forward (previously, to avoid having to do a new PR against against mesosphere/universe). 

(moving tags does seem rude, so maybe we should avoid it in general.)

the practice, as evidenced by other packages in universe, seems to be all over the place. most packages don't use the Image ID; a few packages even use inconstant `latest`. we're fine with dropping `latest` and going through PRs to roll out new versions, and even packaging with the image IDs (we would probably rely on our own universe server to support hot-fixes).

i'd appreciate your suggestion. thanks.